src/app/pages/profile-page/edit/edit.html

```diff
@@ -26,34 +19,41 @@
             <ion-list-header>Profile</ion-list-header>
             <ion-item>
                 <ion-label position="floating" color="primary">Name</ion-label>
-                <ion-input data-test="name-input" type="text" formControlName="name" (ionInput)="onNameChange($event)" [value]="getName()" required></ion-input>
+                <ion-input data-test="name-input" type="text" formControlName="name" (ionInput)="onNameChange($event)"
```

Comment:
Be wary of formatting changes like these.. for instance, when no code changes, just whitespace, tabs, etc. those changes can distract from the real purpose of your pull request.